src/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from './commands';

```

Comment:
I think this file should not export everything.

For example, with `export * from './recommended-checks';`, I think the `beforeEach` / `afterEach` would be executed when someone only wants to import constants.

That is why I would recommend not having a main file.
Users should always imports things specifically.